From 3e148b53eda57ecc66f906ad01c7ae28ae59bd29 Mon Sep 17 00:00:00 2001
From: Nicolas Dufresne <nicolas.dufresne@collabora.com>
Date: Sat, 1 Dec 2018 20:41:10 -0500
Subject: [PATCH 23/45] gl: window-gbm: Remove unused private class member

The display class member is not used, not set and rather
confusing.

(cherry picked from commit 46b503d6749b3c2252be394e302b1dbb6a5934df)
Signed-off-by: Jeffy Chen <jeffy.chen@rock-chips.com>
---
 gst-libs/gst/gl/gbm/gstglwindow_gbm_egl.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/gst-libs/gst/gl/gbm/gstglwindow_gbm_egl.h b/gst-libs/gst/gl/gbm/gstglwindow_gbm_egl.h
index e89af2f537..c27b8c0cf8 100644
--- a/gst-libs/gst/gl/gbm/gstglwindow_gbm_egl.h
+++ b/gst-libs/gst/gl/gbm/gstglwindow_gbm_egl.h
@@ -46,6 +46,8 @@ struct _GstGLWindowGBMEGL {
   struct gbm_bo *current_bo, *prev_bo;
   int waiting_for_flip;
 
+  drmModeCrtc *saved_crtc;
+
   GstGLDisplayGBM *display;
 
   gpointer _reserved[GST_PADDING];
-- 
2.11.0

